beacon-chain/sync/rpc_goodbye.go

```diff
@@ -38,7 +38,7 @@ func (r *Service) goodbyeRPCHandler(ctx context.Context, msg interface{}, stream
 		return fmt.Errorf("wrong message type for goodbye, got %T, wanted *uint64", msg)
 	}
 	log := log.WithField("Reason", goodbyeMessage(*m))
-	log.WithField("peer", stream.Conn().RemotePeer()).Info("Peer has sent a goodbye message")
+	log.WithField("peer", stream.Conn().RemotePeer()).Debug("Peer has sent a goodbye message")
```

Comment:
currently it takes so long to get peers for me. if i just see them connect and don't see that they disconnected i won't understand what is happening and why my node doesn't sync already

Reply:
We have close to 100 goodbye logs in a minute, we should definitely not put it into a `Info` log. This will spam the node. If you need to debug further you can change the verbosity of it.

Reply:
sg